packages/form/core/form-group.ts

```diff
@@ -1,18 +1,24 @@
 import { ControlConfig, FormControl } from './form-control';
 import ShortUniqueId from 'short-unique-id';
 
+export interface FormConfig {
+	name?: string;
+	validateOnLoad?: boolean;
+}
+
 export class FormGroup {
 	controls: FormControl[];
 	name?: string;
 	id?: string;
 
-	constructor(controls: ControlConfig[], name = '') {
+	constructor(controls: ControlConfig[], formConfig: FormConfig = {}) {
```

Comment:
Since the FormGroup constructor is the main API that developers will interact with, I suggest we keep it simple and aim to have no/minimal changes to it! See if we can skip all changes in this class by looping through `controls` in `Form.astro` (see previous comment)